js/src/util.js

```diff
@@ -64,7 +75,7 @@ const Util = (() => {
     },
 
     supportsTransitionEnd() {
-      return Boolean(transition)
+      return Boolean(EventHandler.getBrowserTransitionEnd())
```

Comment:
Not a big fan here because `EventHandler.getBrowserTransitionEnd()` will be called each time `Util.supportsTransitionEnd()` is called